packages/components/src/DateTimeInput.tsx

```diff
@@ -35,13 +36,6 @@ export function fixIncompleteValue(value: string): string {
   return value;
 }
 
-export function addSeparators(value: string): string {
-  const dateTimeMillis = value.substring(0, 23);
-  const micros = value.substring(23, 26);
-  const nanos = value.substring(26);
-  return [dateTimeMillis, micros, nanos].filter(v => v !== '').join('\u200B');
-}
-
 const removeSeparators = (value: string) => value.replace(/\u200B/g, '');
```

Comment:
Mildly inconsistent - why'd we define this a `const` instead of a `function` ?

Reply:
Idk that's just how it was. Probably because it was a single line lambda function so it's less code?